Frameworks/CoreText/CTFont.mm

```diff
@@ -692,13 +704,50 @@ CGGlyph CTFontGetGlyphWithName(CTFontRef font, CFStringRef glyphName) {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes  kCTFontVerticalOrientation is unsupported, returns the same as default and horizontal orientations
 */
 CGRect CTFontGetBoundingRectsForGlyphs(
     CTFontRef font, CTFontOrientation orientation, const CGGlyph glyphs[], CGRect* boundingRects, CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (!glyphs || count <= 0) {
+        TraceError(g_logTag, L"No glyphs were specified");
+        return CGRectNull;
+    }
```

Comment:
yeah I don't think it was this function I meant to comment on. Most of my comments seem to be misplaced / recalled / otherwise not as I intended.... 

---
In reply to: [84556614](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r84556614) [](ancestors = 84556614,84556247,84415484,83340871)